.github/actionlint.yaml

```diff
@@ -0,0 +1,4 @@
+self-hosted-runner:
```

Comment:
Do we actually use a self-hosted runner at all?

Reply:
Yes! We need this for Kubernetes changes. 
Remember that we whitelist source IPs.

Reply:
Oh?  I didn't connect the dots for that one. Will pick your brain about that later.